### ISSUE TITLE
Organizational accounts

### ISSUE THREAD

**Issue by peterthomassen** [desec-io/desec-stack#858]
Allow users to belong to organizations, authorized representatives of which can act in their name (= inspect/manipulate their domains).

**Comment by peterthomassen** [comment]
Design plan: #885, #886, #887, #888

This is closer to token sharing than to hierarchical accounts, so it works in "both directions" if needed. It also comes with scoping capabilities that may be pretty granular if needed.

**Comment by AlperShal** [comment]
Yes, this is really a necessity for a DNS hosting. Lots of times there more than one people managing a domain's DNS entries and settings. It would be nice to not being forced to use one account.
